utils/update_checkout/update_checkout/update_checkout.py

```diff
@@ -371,6 +398,12 @@ def update_all_repositories(args, config, scheme_name, scheme_map, cross_repos_p
                    cross_repos_pr]
         pool_args.append(my_args)
 
+    locked_repositories: set[str] = _is_any_repository_locked(pool_args)
+    if len(locked_repositories) > 0:
+        return [
+            f"'{repo_name}' is locked by git. Cannot update it."
```

Comment:
in what exit status does this result?

Reply:
This adds an error to the `results` variable, which is passed to the `check_parallel_results` function. The exit status is `1`.

```
./swift/utils/update-checkout --scheme release/6.2.1 --skip-repository swift || echo $?
+ pushd /Users/charleszablit/Developer/lldb/swift
+ popd
Skipping update of 'swift', requested by user
======UPDATE FAILURES======
'llvm-project' is locked by git. Cannot update it.
update-checkout failed, fix errors and try again
1
```